Model/AclCacheInterface.php

```diff
@@ -35,11 +35,11 @@ public function evictFromCacheByIdentity(ObjectIdentityInterface $oid);
     /**
      * Retrieves an ACL for the given object identity primary key from the cache.
      *
-     * @param int $primaryKey
+     * @param int $aclId
```

Comment:
the type looks weird to me. `evictFromCacheById` uses `@param string $aclId` while this one uses `@param int $aclId`. Either there is a type issue or they are not the same concept and then it might be better to use different argument names.

Also, the phpdoc still talks about primary key.

Reply:
Hmm, indeed. I can't follow what happens here:

* `evictFromCacheById()` is never used in this package. But the cache implementations in this package all expect a `string` and not an `int`.
* `getFromCacheById()` is used once in this package. But in that case, it seems like an `AclInterface` instance is passed: https://github.com/symfony/security-acl/blob/eb1ddd923910a32635dedb30ececf338a29ba54b/Domain/AclCacheTrait.php#L32-L35

Reply:
`getFromCacheById()` is used once in this package, and `AclInterface::getId()` is passed (which is documented as being an `int`).

However, it's the same acl id as far as I can see.